oem/ibm/configurations/bios/enum_attrs.json

```diff
@@ -375,7 +376,8 @@
             "possible_values": ["Disabled", "Enabled"],
             "default_values": ["Enabled"],
             "helpText": "Only change this value if instructed by service provider as it might degrade system performance. Specifies if Lateral Cast Out mode is disabled or enabled. Do not set this attribute directly; set hb_lateral_cast_out_mode instead.",
-            "displayName": "Lateral Cast Out mode (current)"
+            "displayName": "Lateral Cast Out mode (current)",
+            "readOnly": true
```

Comment:
Why this attribute is made readonly?

Reply:
it's a _current attribute, which can be changed only by HOST and not by any redfish client.